examples/cluster/plot_kmeans_digits.py

```diff
@@ -23,72 +22,136 @@
 AMI          adjusted mutual information
 silhouette   silhouette coefficient
 =========== ========================================================
-
 """
 print(__doc__)
 
-from time import time
+# %%
+# Load the dataset
+# ----------------
+#
+# We will start by loading the `digits` dataset. This dataset contains
+# handwritten digits from 0 to 9. In the context of clustering, one would like
+# to group images such that the handwritten digits on the image are the same.
+
 import numpy as np
-import matplotlib.pyplot as plt
+from sklearn.datasets import load_digits
 
+data, labels = load_digits(return_X_y=True)
+(n_samples, n_features), n_digits = data.shape, np.unique(labels).size
+
+print(
+    f"# digits: {n_digits} \t "
+    f"# samples: {n_samples} \t "
+    f"# features {n_features}"
```

Comment:
Do we want newline here? currently, this looks like:

![Screen Shot 2020-11-12 at 7 13 35 PM](https://user-images.githubusercontent.com/5402633/99011932-38124c00-251b-11eb-8ca0-9b921f71fce0.png)

Reply:
The new lines do not do anything. This the `\t` which tabulate. They were part of the example at first. I can remove them